docs/getting-started/quickstart.md

```diff
@@ -19,7 +19,7 @@ Learn how to set up Semgrep, scan your first project for security issues, and vi
 You must have Python 3.8 or later installed on the machine where the Semgrep CLI is running.
 :::
 
-1. Navigate to [Semgrep Cloud Platform (SCP)](https://semgrep.dev/login), and sign up by clicking on **Sign in with GitHub** or **Sign in with GitLab**. Follow the on-screen prompts to [grant Semgrep the needed permissions](/deployment/checklist/#permissions) and proceed.
+1. Navigate to [Semgrep AppSec Platform (Semgrep AppSec Platform)](https://semgrep.dev/login), and sign up by clicking on **Sign in with GitHub** or **Sign in with GitLab**. Follow the on-screen prompts to [grant Semgrep the needed permissions](/deployment/checklist/#permissions) and proceed.
```

Comment:
```suggestion
1. Navigate to [Semgrep AppSec Platform (SASP)](https://semgrep.dev/login), and sign up by clicking on **Sign in with GitHub** or **Sign in with GitLab**. Follow the on-screen prompts to [grant Semgrep the needed permissions](/deployment/checklist/#permissions) and proceed.
```
This is why humans still have to look at these things

Reply:
Yes, thank you!

Reply:
@s-santillan can you add this change to the commit you’ll push to this branch, depending on our decision about it?

Reply:
Yep, I'll test out "Semgrep" too, it may not always work.